packages/janus-simple-command/package.json

```diff
@@ -62,5 +63,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "ws": "^8.12.1"
```

Comment:
Since the library itself is connection agnostic, we will want a separate package like `jsp-ws-connection` that can be imported and require ws.

You should be able to use the `yarn new-lib --publish jsp-ws-connection` command to create one.